guides/branch-preview.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: page
+title: Branch Preview Mode for UI Modules
+permalink: /guides/branch-preview/
+menuInclude: no
+menuTopTitle: Guides
+---
+
+
+# Branch Preview Mode for UI Modules
+
+Branch preview mode allows developers, product owners, and other interested parties to "preview"
+changes to FOLIO UI components on a live FOLIO system before commmitting them to the master 
+branch. 
+
+## How it works
+
+A FOLIO backend system is built daily from the master branch of the following FOLIO 'platform'
+Github repositories:
+
+https://github.com/folio-org/platform-core (The FOLIO "plaform-core" distribution)  
+https://github.com/folio-org/platform-complete (The FOLIO "platform-complete" distribution)  
+
+These systems contain only the latest releases of Okapi and backend FOLIO modules that are 
+included in either of the FOLIO platforms listed above. 
+
+When a Github PR is opened for an existing branch on,  the following "preview" related
+processes occur in addition to existing quality gates (unit tests, Sonarqube, etc). 
+
+* Code from the UI module's branch is merged into a either platform-core or platform-complete. 
+Essentially replacing the released version of the module.  
+
+* A stripes "bundle" or webpack is compiled and a module descriptor for the module that
+is getting previewed is posted to Okapi on the backend FOLIO instance. 
+
+* An interface dependency check is conducted via Okapi to ensure all required interfaces
+can be resolved to the existing set of released modules deployed on the FOLIO server. 
+
+* A tenant is created on the backend FOLIO system for each build iteration of the PR. The tenant
+naming convention is 'GITHUB_PROJECT NAME_ + PR_NUMBER_ + JENKINS_BUILD_NUMBER'.  
+PR 64 for ui-users would create a tenant called 'ui_users_64_1', for example.  The tenant
+admin user for the tenant is the same as the tenant name + 'admin'.  For example, the tenant
+admin user for tenant, ui_users_64_1, would be 'ui_users_64_1_admin'.  The tenant admin password
+in all cases is 'admin'.  
+
+* Tenant modules are enabled and reference and sample data for select modules are loaded 
+for the tenant. 
+
+* The stripes webpack is distributed to an S3 web service.  An AWS S3 URL to the UI is provided.  
```

Comment:
Whoa!  This is nice.  One clarification, for _"The stripes webpack is distributed to an S3 web service. An AWS S3 URL to the UI is provided."_, how/where is the URL to the UI provided?

Reply:
Jenkins appends it to the PR. Enhanced the doc for that. Thanks.